server/cpho/views/upload_indicators.py

```diff
@@ -63,9 +62,13 @@ def handle_indicators(self, datum):
 
     def handle_indicator_data(self, indicator_obj, datum):
         mapper = upload_mapper()
-        preiod_val = Period.objects.get(
-            year=2021, quarter=None, year_type="calendar"
+        default_period = (
+            Period.relevant_years().filter(year_type="calendar").first()
         )
+        period_val = default_period
+
+        if "Period" in datum and datum["Period"] != "":
+            period_val = mapper["period_mapper"][datum["Period"]]
```

Comment:
If we're gonna support multiple-periods we should probably force everyone to use the column, especially since it saves records automatically without showing them what it does beforehand. So let's not offer a default at all